lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -10,9 +10,13 @@ const ReactDom = require('react-dom');
 const Modal = require('dosomething-modal');
 
 function getLocalStorageKey(user, campaign) {
+  // @TODO: probably need a better way to get the run ID consistently.
+  const languageCode = (Object.keys(campaign.campaign_runs.current)[0]);
+
   let key = null;
+
   if (campaign && user) {
-    key = `onboarding:campaign=${campaign.id}-user=${user.info.id}`;
+    key = `onboarding:campaign=${campaign.id}&run=${campaign.campaign_runs.current[languageCode].id}${user.info ? "&user=" + user.info.id :  ''}`;
```

Comment:
Well, if the campaign happens again and user hasn't cleared and different run ID then no onboarding then... but maybe that's a non-issue...